.vsts-ci.yml

```diff
@@ -3,14 +3,18 @@ trigger:
 - release/*
 
 variables:
-  teamName: Roslyn-Project-System
+  - name: teamName
+    value: Roslyn-Project-System
   ${{ if eq(variables['System.TeamProject'], 'public') }}:
-    PB_PublishBlobFeedKey:
-    PB_PublishBlobFeedUrl:
-    _DotNetPublishToBlobFeed: false
+    - name: PB_PublishBlobFeedUrl
```

Comment:
nit:  For variables defined using array syntax, you could just define the default and then override with the internal condition.  ie.

```yml
variables:
- name: teamName
  value: Roslyn-Project-System
- name: PB_PublishBlobFeedUrl
  value: ''
- ${{ if ne(variables['System.TeamProject'], 'public') }}:
  - name: PB_PublishBlobFeedUrl
    value: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
  - name: _DotNetPublishToBlobFeed
    value: true
```

Reply:
Done.